ddtrace/llmobs/_llmobs.py

```diff
@@ -321,8 +326,10 @@ def _llmobs_span_event(self, span: Span) -> LLMObsSpanEvent:
             try:
                 llmobs_span._tags = cast(Dict[str, str], span._get_ctx_item(TAGS))
                 user_llmobs_span = self._user_span_processor(llmobs_span)
+                if user_llmobs_span is None:
+                    return None
```

Comment:
i think we should also add a `span_event is not None` check [here](https://github.com/DataDog/dd-trace-py/blob/280be7679757b720503a284a69da36995b6ef48d/ddtrace/llmobs/_llmobs.py#L232-L233)

Reply:
it should be there? https://github.com/DataDog/dd-trace-py/blob/e16a791a8c18ff805d835fbf0d7fc133f7b4995f/ddtrace/llmobs/_llmobs.py#L236-L237